test/GasUsage.test.js

```diff
@@ -39,4 +39,15 @@ describe('ERC721A Gas Usage', function () {
       }
     });
   });
+
+  context('transferFrom', function () {
+    it('transfer to and fro', async function () {
```

Comment:
Typo: "from"

Reply:
Ok, changed to "transfer to and from two addresses"